### PR TITLE
fix mssql security context for ocp4.11

### PR DESCRIPTION
**Pull request by ebattat**
This PR fix MSSQL security context issue for OCP4.11
There are 2 changes:
1. Add to the [mssql.yaml](https://github.com/redhat-performance/benchmark-runner/blob/main/benchmark_runner/common/template_operations/templates/hammerdb/internal_data/mssql_template.yaml)
```
        securityContext:
          allowPrivilegeEscalation: true
```
2. Update admin for mssql-db namespace:
```
oc adm policy add-scc-to-group restricted system:serviceaccounts --context admin mssql-db
OR
oc adm policy add-scc-to-group restricted system:authenticated --context admin mssql-db
```

Also have opened [issue](https://github.com/microsoft/mssql-docker/issues/769) for mssql container team for getting the correct security context for image: mcr.microsoft.com/mssql/rhel/server:2019-latest

### CODE REVIEW

File: benchmark_runner/common/ocp_resources/custom/template/01_coredns_patch_template.sh [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # solved Error: Failed to download metadata for repo 'fedora-cisco-openh264': Cannot prepare internal mirrorlist
 oc get nodes -l node-role.kubernetes.io/worker= -o jsonpath="{range .items[*]}{.metadata.name}{'\n'}{end}" |  xargs -I{} oc debug node/{} -- chroot /host sh -c "sed -i '/bufsize 512/d' /etc/coredns/Corefile"
+
```
Comment (RobertKrawitz) [inline benchmark_runner/common/ocp_resources/custom/template/01_coredns_patch_template.sh:3]: Not needed -- not harmful, but unnecessary change.

File: benchmark_runner/common/oc/oc.py [hunk 2]
```diff
@@ -598,7 +598,7 @@ def wait_for_ready(self, label: str, run_type: str = 'pod', workload: str = '',
             if 'met' in result.decode("utf-8"):
                 return True
         except Exception as err:
-            if 'pod' in workload:
+            if 'vm' not in workload:
```
Comment (RobertKrawitz) [inline benchmark_runner/common/oc/oc.py:601]: Suggest doing this the way you did the previous change:

```
            if 'vm' in workload:
                raise VMNotReadyTimeout(workload=workload)
            else:
                raise PodNotReadyTimeout(workload=workload)
```

just in case we wind up with a workload whose name actually contains 'pod'.  Not a major issue.

**Review by RobertKrawitz — COMMENTED** [review]
Very minor issues, if you don't want to change them, I'll approve it as is.

**Comment by pjgg** [comment]
Hi @RobertKrawitz, do you know if this patch is already included on OpenShift 4.11 ?

**Comment by ebattat** [comment]
Hi @pjgg, 
OCP 4.11 contains permissions/ securityContext changes.
Meaning we need to use this fix till we get the right security context of [mssql](https://github.com/microsoft/mssql-docker/issues/769)
